origin_ui/advertise.go

```diff
@@ -60,10 +60,31 @@ func AdvertiseOrigin() error {
 	// TODO: waiting on a different branch to merge origin URL generation
 	originUrl := "https://localhost:8444"
 
+	// Here we instantiate the namespaceAd slice, but we still need to define the namespace
+	namespaceUrl, err := url.Parse(viper.GetString("NamespaceUrl"))
+	if err != nil {
+		return errors.New("Bad namespaceUrl")
+	}
+	if namespaceUrl.String() == "" {
+		return errors.New("No NamespaceUrl is set")
+	}
+
+	prefix := viper.GetString("NamespacePrefix")
+
+	// TODO: Need to figure out where to get some of these values
+	// 		 so that they aren't hardcoded...
+	nsAd := director.NamespaceAd{
+		RequireToken:  false,
```

Comment:
```suggestion
		RequireToken:  true,
```

even though these are mostly placeholders before we improve it next version, I want to default to authorization...